settings/_settings.global.scss

```diff
@@ -23,10 +29,13 @@ $inuit-global-transition:   (1/3) + s !default;
 // Spacing values are determined based on your project’s global line height (i.e
 // your baseline grid). It is not recommended that you modify these following
 // variables (it can break your vertical rhythm), but if you need to, you can.
+
 $inuit-global-spacing-unit: round($inuit-global-line-height) !default;
 
+
 // How many times larger/smaller than the default should our spacing unit
 // variants be?
+
 $inuit-global-spacing-unit-factor-tiny:   0.25 !default;
 $inuit-global-spacing-unit-factor-small:  0.5  !default;
 $inuit-global-spacing-unit-factor-large:  2    !default;
```

Comment:
Just noticed: The whitespace between variable declarations and values are very inconsistent in the whole file here. Personally, I think we should add only one space between colon and values everywhere.